docs/en/observability/cloud-monitoring/aws/monitor-aws-esf.asciidoc

```diff
@@ -0,0 +1,149 @@
+2. Select the the network interface you want to use.
+3. From the *Actions* drop-down menu, choose *Create flow log*.
+4. For *Destination*, select *Send to an S3 bucket*.
+5. For *S3 bucket ARN*, entre the name of the S3 bucket you created in the previous step. 
+
+For more details, refer to the Amazon documentation on how to https://docs.aws.amazon.com/vpc/latest/userguide/flow-logs-s3.html[Create a flow log that publishes to Amazon S3].
+
+[discrete]
+[[esf-step-three]]
+=== Step 3: Create an SQS queue and notifications for VPC flow logs
+
+The Amazon Simple Queue Service (SQS) event notification on Amazon S3 serves as a trigger for the Lambda function. When a new log file gets written to an Amazon S3 bucket and meets the criteria, a notification is generated that triggers the Lambda function.
+
+1. In the https://console.aws.amazon.com/sqs/[SQS console] create a standard SQS queue that uses the default settings.
+2. Edit the queue you created to define an advanced access policy. The access policy allows S3 ObjectCreated events to be sent to the queue.
+3. Go to the properties of the S3 bucket containing the VPC flow logs and enable event notification.
+
+For more details, refer to the AWS documentation on how to https://docs.aws.amazon.com/AmazonS3/latest/userguide/ways-to-add-notification-config-to-bucket.html[Configure a bucket for notifications].
+
+[discrete]
+[[esf-step-four]]
+=== Step 4: Install the Elastic AWS integration 
+
+{kib} offers prebuilt dashboards, ingest node configurations, and other assets that help you get the most value out of the logs you ingest. 
+
+1. Go to *Integrations* in {kib} and search for AWS. 
+2. Click the AWS integration, select *Settings* and click *Install AWS assets* to install all the AWS integration assets.
+
+*TBD* == we might need extra details here
+
+[discrete]
+[[esf-step-five]]
+=== Step 5: Create a new S3 bucket
+
+You create a new S3 bucket and queue for the access logs, then configure the older S3 bucket to generate access logs.
+
+1. In the https://s3.console.aws.amazon.com/s3[AWS S3 console], click *Create* bucket. Give the bucket a name and specify the region where you want it deployed.
++
+IMPORTANT: Make sure you create the S3 bucket and SQS queue (next step) in the same region as the bucket containing VPC flow logs.
+
+2. Follow the steps you learned earlier to create an SQS queue and edit the access policy (use the ARNs of the new S3 bucket and queue). Make a note of the queue URL because you will need it later when you configure S3 access log collection.
+
+3. Configure the new S3 bucket to send notifications to the new queue when objects are created (follow the steps you learned earlier).
+
+4. Go back to the old S3 bucket (the one that contains VPC flow logs), and under Properties, edit the Server access logging properties. Enable server access logging, and select the new bucket you created as the target bucket.
```

Comment:
To set up ESF for the first time, users have to perform two actions:

1. Create an S3 bucket to store the configuration file.
2. Create the actual configuration file.

We can keep these two actions in steps 5 and 6 in this new guide, but we need to update the content.

I can share the steps here or discuss them over Zoom and update the PR afterward.

Reply:
Thanks for your help, @zmoog 
I scheduled a time slot for us to fix this.

Reply:
@zmoog After the meeting we had yesterday, this section has been completely rewritten.